common/src/test/scala/io/phdata/test/fixtures/package.scala

```diff
@@ -51,7 +51,7 @@ package object fixtures {
   val initialLDAP = savedLDAP.copy(id = None)
   val savedGrant = HiveGrant("sw_sesame", "/shared_workspaces/sw_sesame", savedLDAP, Manager, id = Some(id))
   val initialGrant = savedGrant.copy(id = None, ldapRegistration = initialLDAP)
-  val savedHive = HiveAllocation("sw_sesame", "/shared_workspaces/sw_sesame", hdfsRequestedSize, None, savedGrant, readonlyGroup = Some(savedGrant.copy(databaseRole = ReadOnly)), id = Some(id))
+  val savedHive = HiveAllocation("sw_sesame", "hdfs://shared_workspaces/sw_sesame", hdfsRequestedSize, None, savedGrant, readonlyGroup = Some(savedGrant.copy(databaseRole = ReadOnly)), id = Some(id))
```

Comment:
I think we should test if there is not scheme `xxx://` then we treat it as hdfs, so `/foo/bar` is treated as hdfs just like `hdfs://foo/bar` .

The reason is that Hive will default to HDFS if there is no scheme, so we should allow it.